Bottom: 66a7fcb5bbf404ae743ee1940c00a1089e175876
Top:    7cbcb3a4819f129035cfe035e3cf2e7f0685e998
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-14 02:45:32 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 507aa78..5a44fb8 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
